licenses/VXL

```diff
@@ -0,0 +1,30 @@
+Copyright 2000-2013 VXL Contributors
```

Comment:
The only vxl license I see is the one I added myself :-/ amy I missing anything?

```
chymera@zenbookhost ~ $ ls /usr/portage/licenses/V*
/usr/portage/licenses/ValveSDK    /usr/portage/licenses/Vivaldi
/usr/portage/licenses/VGBA        /usr/portage/licenses/Volano
/usr/portage/licenses/Vidyo-EULA  /usr/portage/licenses/VOSTROM
/usr/portage/licenses/VIGRA       /usr/portage/licenses/VTK
chymera@zenbookhost ~ $ ls /usr/portage/licenses/v*
/usr/portage/licenses/ventrilo    /usr/portage/licenses/vlgothic
/usr/portage/licenses/vienna-rna  /usr/portage/licenses/vmd
/usr/portage/licenses/vim         /usr/portage/licenses/vmware
/usr/portage/licenses/vim.org
chymera@zenbookhost ~ $ ls /home/chymera/src/neurogentoo/licenses/V*
/home/chymera/src/neurogentoo/licenses/VXL
chymera@zenbookhost ~ $ ls /home/chymera/src/neurogentoo/licenses/v*
/home/chymera/src/neurogentoo/licenses/vampir
```